### ISSUE TITLE
Build failure: ente-auth

### ISSUE THREAD

**Issue by ryand56** [NixOS/nixpkgs#352616]
### Steps To Reproduce

1. build *ente-auth*

### Build log

```
strip RPath of /nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/ente-auth/lib/libfile_saver_plugin.so
/nix/store/fpl6ms6bfp0017akvs0hn5wxr85ih21j-at-spi2-core-2.52.0/lib:/nix/store/ms8np5xp0l9g04nrxhv6kh7cqcljyvlw-cairo-1.18.2/lib:/nix/store/p054qzi353lvvbwbvibgr2934np50z03-gdk-pixbuf-2.42.12/lib:/nix/store/fxnchbq09sq279ssqdsgzigsyvp6hz06-glib-2.80.4/lib:/nix/store/z49s72m05k5daqmv7pajncx55v4p8fnm-gtk+3-3.24.43/lib:/nix/store/ckmsqq30bk7i7ykkl7dydcjj609h67pr-harfbuzz-9.0.0/lib:/nix/store/v6vln32dqlsiiq5l2bm3wxkbg95ladn2-libepoxy-1.5.10/lib:/nix/store/mmc6kxqdavjq41zpxhsnlf19kl6m6cri-pango-1.52.2/lib:/nix/store/nicg80r560lxr1vzlrjcj401v98fwb7y-libX11-1.8.10/lib:/nix/store/nalbkvf08jrg0x2qb73218rnzgmk9nvi-libdeflate-1.21/lib:/build/source/auth/linux/flutter/ephemeral:/nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/lib:/nix/store/haci0d018njygwdga95bsbx8ww2iswy7-zlib-1.3.1/lib:/nix/store/sl141d1g77wvhr050ah87lcyz2czdxa3-glibc-2.40-36/lib:/nix/store/y3kdn61k93rq2jx1lj2x72lnsk0l92qh-gcc-13.3.0-lib/lib:/nix/store/lw294qfdnf29lhxqwzy9byrmx975rvn9-gcc-13.3.0-libgcc/lib
strip RPath of /nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/ente-auth/lib/libscreen_retriever_plugin.so
/nix/store/fpl6ms6bfp0017akvs0hn5wxr85ih21j-at-spi2-core-2.52.0/lib:/nix/store/ms8np5xp0l9g04nrxhv6kh7cqcljyvlw-cairo-1.18.2/lib:/nix/store/p054qzi353lvvbwbvibgr2934np50z03-gdk-pixbuf-2.42.12/lib:/nix/store/fxnchbq09sq279ssqdsgzigsyvp6hz06-glib-2.80.4/lib:/nix/store/z49s72m05k5daqmv7pajncx55v4p8fnm-gtk+3-3.24.43/lib:/nix/store/ckmsqq30bk7i7ykkl7dydcjj609h67pr-harfbuzz-9.0.0/lib:/nix/store/v6vln32dqlsiiq5l2bm3wxkbg95ladn2-libepoxy-1.5.10/lib:/nix/store/mmc6kxqdavjq41zpxhsnlf19kl6m6cri-pango-1.52.2/lib:/nix/store/nicg80r560lxr1vzlrjcj401v98fwb7y-libX11-1.8.10/lib:/nix/store/nalbkvf08jrg0x2qb73218rnzgmk9nvi-libdeflate-1.21/lib:/build/source/auth/linux/flutter/ephemeral:/nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/lib:/nix/store/haci0d018njygwdga95bsbx8ww2iswy7-zlib-1.3.1/lib:/nix/store/sl141d1g77wvhr050ah87lcyz2czdxa3-glibc-2.40-36/lib:/nix/store/y3kdn61k93rq2jx1lj2x72lnsk0l92qh-gcc-13.3.0-lib/lib:/nix/store/lw294qfdnf29lhxqwzy9byrmx975rvn9-gcc-13.3.0-libgcc/lib
strip RPath of /nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/ente-auth/lib/libgtk_plugin.so
/nix/store/fpl6ms6bfp0017akvs0hn5wxr85ih21j-at-spi2-core-2.52.0/lib:/nix/store/ms8np5xp0l9g04nrxhv6kh7cqcljyvlw-cairo-1.18.2/lib:/nix/store/p054qzi353lvvbwbvibgr2934np50z03-gdk-pixbuf-2.42.12/lib:/nix/store/fxnchbq09sq279ssqdsgzigsyvp6hz06-glib-2.80.4/lib:/nix/store/z49s72m05k5daqmv7pajncx55v4p8fnm-gtk+3-3.24.43/lib:/nix/store/ckmsqq30bk7i7ykkl7dydcjj609h67pr-harfbuzz-9.0.0/lib:/nix/store/v6vln32dqlsiiq5l2bm3wxkbg95ladn2-libepoxy-1.5.10/lib:/nix/store/mmc6kxqdavjq41zpxhsnlf19kl6m6cri-pango-1.52.2/lib:/nix/store/nicg80r560lxr1vzlrjcj401v98fwb7y-libX11-1.8.10/lib:/nix/store/nalbkvf08jrg0x2qb73218rnzgmk9nvi-libdeflate-1.21/lib:/build/source/auth/linux/flutter/ephemeral:/nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/lib:/nix/store/haci0d018njygwdga95bsbx8ww2iswy7-zlib-1.3.1/lib:/nix/store/sl141d1g77wvhr050ah87lcyz2czdxa3-glibc-2.40-36/lib:/nix/store/y3kdn61k93rq2jx1lj2x72lnsk0l92qh-gcc-13.3.0-lib/lib:/nix/store/lw294qfdnf29lhxqwzy9byrmx975rvn9-gcc-13.3.0-libgcc/lib
strip RPath of /nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/ente-auth/lib/libwindow_manager_plugin.so
/nix/store/fpl6ms6bfp0017akvs0hn5wxr85ih21j-at-spi2-core-2.52.0/lib:/nix/store/ms8np5xp0l9g04nrxhv6kh7cqcljyvlw-cairo-1.18.2/lib:/nix/store/p054qzi353lvvbwbvibgr2934np50z03-gdk-pixbuf-2.42.12/lib:/nix/store/fxnchbq09sq279ssqdsgzigsyvp6hz06-glib-2.80.4/lib:/nix/store/z49s72m05k5daqmv7pajncx55v4p8fnm-gtk+3-3.24.43/lib:/nix/store/ckmsqq30bk7i7ykkl7dydcjj609h67pr-harfbuzz-9.0.0/lib:/nix/store/v6vln32dqlsiiq5l2bm3wxkbg95ladn2-libepoxy-1.5.10/lib:/nix/store/mmc6kxqdavjq41zpxhsnlf19kl6m6cri-pango-1.52.2/lib:/nix/store/nicg80r560lxr1vzlrjcj401v98fwb7y-libX11-1.8.10/lib:/nix/store/nalbkvf08jrg0x2qb73218rnzgmk9nvi-libdeflate-1.21/lib:/build/source/auth/linux/flutter/ephemeral:/nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/lib:/nix/store/haci0d018njygwdga95bsbx8ww2iswy7-zlib-1.3.1/lib:/nix/store/sl141d1g77wvhr050ah87lcyz2czdxa3-glibc-2.40-36/lib:/nix/store/y3kdn61k93rq2jx1lj2x72lnsk0l92qh-gcc-13.3.0-lib/lib:/nix/store/lw294qfdnf29lhxqwzy9byrmx975rvn9-gcc-13.3.0-libgcc/lib
strip RPath of /nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/ente-auth/lib/libsentry_flutter_plugin.so
install: cannot stat '/nix/store/f5a3ih6ykf0xjv3p67ks9zvdigjyxnif-ente-auth-4.0.2/app/data/flutter_assets/assets/icons/auth-icon.png': No such file or directory
```

### Notify maintainers

@niklaskorz @schnow265 

### Metadata

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.5-xanmod1, NixOS, 24.11 (Vicuna), 24.11.20241025.18536bf`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.0
```
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by chewblacka** [comment]
I think this might be a `lix` issue. It builds fine for me using `nix`. The store path is:

    /nix/store/jb55xa5y1asn8crbj5g8m0zh1pgmshcb-ente-auth-4.0.2

**Comment by niklaskorz** [comment]
Fixed by #352749
